src/runge-kutta.jl

```diff
@@ -46,8 +46,19 @@ name(stepper::RKStepper) = stepper.tableau.name
 isadaptive(::RKStepper{:adaptive}) = true
 isadaptive(::RKStepper{:fixed}) = false
 
-solve{T,S<:RKStepper}(ode::ExplicitODE{T}, stepper::Type{S}; options...) =
-    Solver(ode,stepper{T}(;options...))
+function solve{T,S<:RKStepper}(ode::ExplicitODE{T},
+                               ::Type{S};
+                               tspan = T[Inf],
+                               tstop = tspan[end],
+                               options...)
+    t0 = ode.t0
+    if tstop < t0
+        tspan = 2t0.-tspan
```

Comment:
if `tspan = [t0,t0+dt]` (with `dt<0`), then an interval of the same length and starting at `t0`, but reversed, would be `[t0,t0-dt]`.  But `dt=tstop-t0` and the same for `tspan`.